packages/terra-clinical-item-display/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ ChangeLog
 
 Unreleased
 ----------
+### Breaking Change
```

Comment:
This should no longer be a breaking change right given this [discusison](https://github.com/cerner/terra-clinical/pull/555/files#r320387476)?

Reply:
Updated here https://github.com/cerner/terra-clinical/pull/555/commits/8a49744939b56679d9238896c0fa6b5051e12e6d.